EventCpu.hpp:284:32: warning: declaration of ‘lk’ shadows a previous local [-Wshadow]

```
alpaka/include/alpaka/event/EventCpu.hpp:284:32: warning: declaration of ‘lk’ shadows a previous local [-Wshadow]
                                 std::lock_guard<std::mutex> lk(spEventCpuImpl->m_Mutex);
                                ^
alpaka/include/alpaka/event/EventCpu.hpp:248:30: note: shadowed declaration is here
                     std::lock_guard<std::mutex> lk(spEventCpuImpl->m_Mutex);
                              ^
```
As this gets repeated on every object compilation, I find it crucial to eliminate these warnings somehow, either through cmake or by renaming the second `lk` inside the lambda with `lk2` or some more meaningful name even.

--- a/include/alpaka/event/EventCpu.hpp
+++ b/include/alpaka/event/EventCpu.hpp
@@ -281,7 +281,7 @@ namespace alpaka
                         [spEventCpuImpl]()
                         {
                             {
-                                std::lock_guard<std::mutex> lk(spEventCpuImpl->m_Mutex);
+                                std::lock_guard<std::mutex> lk2(spEventCpuImpl->m_Mutex);
                                 // Nothing to do if it has been re-enqueued to a later position in the queue.
                                 if(spEventCpuImpl->m_canceledEnqueueCount > 0)
                                 {

